library/src/schemas/any/any.ts

```diff
@@ -5,7 +5,11 @@ import { executePipe } from '../../utils/index.ts';
  * Any schema type.
  */
 export type AnySchema<TOutput = any> = BaseSchema<any, TOutput> & {
-  schema: 'any';
+  kind: 'any';
+  /**
+   * Validation and transformation pipe.
```

Comment:
For types, I don't think these inline comments are necessary.

Reply:
As a result of your comment, I have changed my mind on this. Thank you for this improvement. However, I would prefer to add a blank line before a comment that starts with `/**` for a better overview.

Reply:
I'll make a pass over everything tonight to try and clean up the formatting.